src/ray/core_worker/context.h

```diff
@@ -171,6 +172,10 @@ class WorkerContext {
 
   /// Per-thread worker context.
   static thread_local std::unique_ptr<WorkerThreadContext> thread_context_;
+
+  /// Maps from serialized runtime env to its parsed json.
+  ::ray::utils::container::SharedLruCache<std::string, nlohmann::json>
+      parsed_runtime_env_cache_;
```

Comment:
What's the perf gain from this?

Reply:
Release test continues to fail due to unknown reason

Reply:
Update: it improves throughput from ~80 -> ~580